psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -764,13 +759,72 @@ Vector3 OEProp::compute_center(const double *property) const
     return Vector3(x, y, z);
 }
 
+
+OEProp::OEProp(std::shared_ptr<Wavefunction> wfn) : Prop(wfn), mpc(wfn,get_origin_from_environment()), pac(wfn), epc(wfn)
+{
+    common_init();
+}
+OEProp::~OEProp()
+{
+}
+
 void OEProp::common_init()
 {
-    // See if the user specified the origin
     Options &options = Process::environment.options;
-
     print_ = options.get_int("PRINT");
 
+    // Determine number of NOONs to print; default is 3
+    if(options.get_str("PRINT_NOONS") == "ALL") max_noon_ = wfn_->nmo();
+        else max_noon_ = to_integer(options.get_str("PRINT_NOONS"));
+}
+
+
+MultipolePropCalc::MultipolePropCalc(std::shared_ptr<Wavefunction> wfn, Vector3 const & origin) : Prop(wfn), origin_(origin)
+{
+    typedef MultipolePropCalc MPC;
```

Comment:
This does not appear to be used.

Reply:
True, it is a leftover, which I will remove, while amending the pull request. It was used similar to the PAC typedef to simplify outputs and make everything a bit shorter.